### ISSUE TITLE
Entry point and script are maybe redundant?

### ISSUE THREAD

**Issue by xylar** [cmecmetrics/cmec-driver#8]
@acordonez and @paullric, we're all set on `conda-forge`, as you may have seen me mention. I can do the following:

```
conda config --add channels conda-forge
conda config --set channel_priority strict
conda create -y -n test -c conda-forge/label/cmec_dev python=3.9 cmec_driver
conda activate test
which cmec_driver.py 
which cmec-driver 
cmec-driver --help
cmec_driver.py --help
```

It's a little confusing maybe having both the `cmec-driver` entry point and the `cmec_driver.py` script. Would it make sense just to have the entry point?

Also, let me know when you want to do another alpha or beta release and I can help with the process on conda-forge.

**Comment by acordonez** [comment]
@xylar Thanks for checking the install! It would make sense just to have the entry point - I assume I just remove "scripts=['cmec_driver/cmec_driver.py']" from setup.py next release?

**Comment by xylar** [comment]
Yes, exactly.  That should do it. The file will still be around to be used as a module, it just won't be in a user's execution path.
